diff --git a/src/libext/libext_utils/cmake.sh b/src/libext/libext_utils/cmake.sh
index 451fd11f60..35870a5a65 100644
--- a/src/libext/libext_utils/cmake.sh
+++ b/src/libext/libext_utils/cmake.sh
@@ -8,15 +8,19 @@ get_cmake_release(){
     echo "Parameter #1 is $1"
     echo cmake_instdir is $cmake_instdir
     rm -f cmake-${CMAKE_VER}.tar.gz
-    if [[ ${UNAME_S} == "Linux" ]] && [[ ${CPU} == "x86_64" ||  ${CPU} == "aarch64" || ${CPU} == "i686" ]] ; then
-	cd $cmake_instdir
-	if [[ ${CPU} == "i686" ]] ; then
-	    CMAKE_CPU="x86_64"
+    if [[ ${UNAME_S} == "Linux" ]]; then
+	if [[ ${CPU} == "x86_64" ||  ${CPU} == "aarch64" || ${CPU} == "i686" ]] ; then
+	    cd $cmake_instdir
+	    if [[ ${CPU} == "i686" ]] ; then
+		CMAKE_CPU="x86_64"
+	    else
+		CMAKE_CPU=${CPU}
+	    fi
+	    CMAKE=`pwd`/cmake-${CMAKE_VER}-linux-${CMAKE_CPU}/bin/cmake
+	    CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VER}/cmake-${CMAKE_VER}-linux-${CMAKE_CPU}.tar.gz
 	else
-	    CMAKE_CPU=${CPU}
+	        get_cmake_master
 	fi
-	CMAKE=`pwd`/cmake-${CMAKE_VER}-linux-${CMAKE_CPU}/bin/cmake
-	CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VER}/cmake-${CMAKE_VER}-linux-${CMAKE_CPU}.tar.gz
     elif [[ ${UNAME_S} == "Darwin" ]] ; then
 	cd $cmake_instdir
 	CMAKE=`pwd`/cmake-${CMAKE_VER}-macos-universal/CMake.app/Contents/bin/cmake
