diff --git a/src/util/util_strips.F b/src/util/util_strips.F
index 166b895424..85df671a8c 100644
--- a/src/util/util_strips.F
+++ b/src/util/util_strips.F
@@ -11,9 +11,10 @@ c
       integer jj,acc_ilo,acc_ihi,acc_ilgt
       integer iptr
       integer npass
-#define ACC_LENGTH 800
-#define ACC_LENGTH_MAX 950
-#define ACC_LENGTH_MAX_ 1950
+#define ACC_LENGTH 128
+#define ACC_LENGTH_MAX 160
+#define ACC_LENGTH_ 800
+#define ACC_LENGTH_MAX_ 950
 c
       if(lgth_in.gt.0) then
 c         npass=(ld*(jhi-jlo+1))/max(lgth_in+1
