diff --git a/src/hessian/analytic/GNUmakefile b/src/hessian/analytic/GNUmakefile
index 5ca8452807..66f2eb2b57 100644
--- a/src/hessian/analytic/GNUmakefile
+++ b/src/hessian/analytic/GNUmakefile
@@ -2,16 +2,17 @@
 
 SUBDIRS = dft 
 
+OBJ =   \
+                onedd_calc.o \
+                oned_calc.o \
+                onedd_cont.o 
 OBJ_OPTIMIZE =  hess_anal.o \
                 hess_init.o \
                 hess_energyinit.o \
                 hess_check.o \
                 nucdd_cont.o \
                 nucdd_calc.o \
-                oned_calc.o \
-                onedd_cont.o \
                 get_wdens.o \
-                onedd_calc.o \
                 hess_dipder.o \
                 twodd_cont.o \
                 twodd_coul_ex.o \
